### ISSUE TITLE
inversion train issue: weird results

### ISSUE THREAD

**Issue by Sakura-ldx** [yuval-alaluf/hyperstyle#30]
When the global step < 15000, the losses are decreasing and images in logs/images/train are natural though they're a bit different from input. Then the losses rises to about 1.0 and images in logs/images/train get weird. I'm confused, hope to get your help.

**Comment by yuval-alaluf** [comment]
I would need some more details to help you understand what could be causing this. Can you provide some details on your parameters and data?

**Comment by Sakura-ldx** [comment]
> I would need some more details to help you understand what could be causing this. Can you provide some details on your parameters and data?

Sorry, I'm busy in collect data these days so that I send this message today. The encoder is a pretrained pSp model trained with our own face data. Some parameters: output_size=8, batch_size=8, lr=1e-4, n_iters_per_batch=5, loss hyperparameters are same as the article.

**Comment by yuval-alaluf** [comment]
The parameters you mention seem fine, but you say the images get weird. What do you mean by this? What kind of data are you working with? What's your domain? If you can provide some example results, this could help me understand what may be causing the issue.

**Comment by Sakura-ldx** [comment]
> The parameters you mention seem fine, but you say the images get weird. What do you mean by this? What kind of data are you working with? What's your domain? If you can provide some example results, this could help me understand what may be causing the issue.
![微信图片_20220402215549](https://user-images.githubusercontent.com/68360726/161386499-c2f6ba3a-8277-4d39-9c17-02c43a6354f7.png)
![微信图片_20220402215613](https://user-images.githubusercontent.com/68360726/161386509-c2dd321b-db3b-4774-ac05-54dd68290a73.png)
The first image is the 1000 step's, the second is the 11000 step's.

**Comment by yuval-alaluf** [comment]
Were any other adjustments made to the code?
